City-of-Pori/pori-kada
# PORI-281 Added visitpori front page context and modified visitpori la…

…nding page context and front page context

To test:

1. drush fra -y && drush cc all
2. Check that visitpori front page elements are in correct order:
  1. karusellinosto
  2. pääuutisnostot
  3. projektinostot
  4. kuvalliset linkkilaatikot
  5. ajankohtaisvirta
3. Check that pori.fi front page elements are still in correct order.
4. Check that visitpori landing pages are in correct order:
  1. karusellinosto
  2. kuvalliset linkkilaatikot
  3. projektinostot
  4. ajankohtaisvirta
